### ISSUE TITLE
unable to pull image: Error determining manifest MIME type

### ISSUE THREAD

**Issue by aglitke** [openshift/installer#1759]
# Version

```console
$ openshift-install version
./bin/openshift-install unreleased-master-991-g3b25c88e2d13c125c7112cacda77ed8c85fedf27
built from commit 3b25c88e2d13c125c7112cacda77ed8c85fedf27
release image registry.svc.ci.openshift.org/origin/release:v4.1
```

# Platform (aws|libvirt|openstack):

libvirt

# What happened?

Bootstrap fails (from `journalctl -b -f -u bootkube.service`):
```
May 17 15:28:55 test1-5lfzl-bootstrap systemd[1]: Started Bootstrap a Kubernetes cluster.
May 17 15:29:20 test1-5lfzl-bootstrap bootkube.sh[23426]: Rendering cluster config manifests...
May 17 15:29:21 test1-5lfzl-bootstrap bootkube.sh[23426]: unable to pull registry.svc.ci.openshift.org/origin/4.1-2019-05-13-204940@sha256:2052e13f8a6e1ce52f9f719b05e0cf4f694079295bd0e6e4aa95c37eab4c0192: unable to pull image: Error determining manifest MIME type for docker://registry.svc.ci.openshift.org/origin/4.1-2019-05-13-204940@sha256:2052e13f8a6e1ce52f9f719b05e0cf4f694079295bd0e6e4aa95c37eab4c0192: Error reading manifest sha256:2052e13f8a6e1ce52f9f719b05e0cf4f694079295bd0e6e4aa95c37eab4c0192 in registry.svc.ci.openshift.org/origin/4.1-2019-05-13-204940: manifest unknown: manifest unknown
May 17 15:29:21 test1-5lfzl-bootstrap systemd[1]: bootkube.service: Main process exited, code=exited, status=125/n/a
May 17 15:29:21 test1-5lfzl-bootstrap systemd[1]: bootkube.service: Failed with result 'exit-code'.
May 17 15:29:26 test1-5lfzl-bootstrap systemd[1]: bootkube.service: Service RestartSec=5s expired, scheduling restart.
```

See [the troubleshooting documentation](https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md) for ideas about what information to collect.
For example, if the installer [fails to create resources](https://github.com/openshift/installer/blob/master/docs/user/troubleshooting.md#installer-fails-to-create-resources), attach the relevant portions of your `.openshift_install.log`.

# What you expected to happen?

Bootstrap completes successfully.

# How to reproduce it (as minimally and precisely as possible)?

```console
$ bin/openshift-install create cluster
```

# Anything else we need to know?

# References

**Comment by abhinavdahiya** [comment]
Fixed by https://github.com/openshift/installer/pull/1756

**Comment by abhinavdahiya** [comment]
> Fixed by #1756

/close
